samples/Go/go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/leighlondon/lice
+
+require (
+	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/pmezard/go-difflib v1.0.0 // indirect
+	github.com/sergi/go-diff v1.0.0
+	github.com/stretchr/testify v1.2.2 // indirect
+)
```

Comment:
Hm, this one is [strangely highlighted too](https://github-lightshow.herokuapp.com/?utf8=%E2%9C%93&scope=source.go&grammar_format=auto&grammar_url=&grammar_text=&code_source=from-url&code_url=https%3A%2F%2Fgithub.com%2Fgithub%2Flinguist%2Fblob%2Fb790a10fcdff41eec39e8d026ed712dbd9693339%2Fsamples%2FGo%2Fgo.mod&code=). Is that not valid Go code?

Reply:
It's not, but it's definitely not AMPL either (which it's currently trying to be highlighted as).